### PR TITLE
[WIP] Canonical tags

### PR DESCRIPTION
**Pull request by johnplayer1982**
Adding front end canonical tags for SEO benefit. Engines changed in relation to this task:

feedback
contribution_calculator
payday_loans_intervention
baby_cost_calculator
timelines

Pipelines for the above have passed, versions updated

### CODE REVIEW

**Comment by jonalport** [comment]
:+1:

**Comment by jonalport** [comment]
Whoops

**Comment by benbarnett** [comment]
What was breaking the tests out of interest?

**Comment by benbarnett** [comment]
:+1:

**Comment by johnplayer1982** [comment]
I removed the set_meta_tags content from the show.html.erb in static_pages and hardcoded the canonical into the en/cy contact us pages as these are the only ones which use the static_page template, however the set_meta_tags section was still expected in show.html.erb, so was failing based on that, I've added this back in.  Editing the canonical url for contact us will be a separate issue as the page resides in /static/ and already has a canonical tag, the request is to change the canonical url to /en/corporate/contact-us.  Routing issue maybe?

File: Gemfile.lock [hunk 1]
```diff
@@ -657,7 +658,7 @@ GEM
     transitions (0.1.13)
     tzinfo (1.2.2)
       thread_safe (~> 0.1)
-    uglifier (2.7.1)
+    uglifier (2.7.2)
```
Comment (rnalexander) [inline Gemfile.lock]: What's making the uglifier update?

File: Gemfile.lock [hunk 2]
```diff
@@ -378,13 +379,13 @@ GEM
       nokogiri (~> 1.6.0)
       ruby_parser (~> 3.5)
     html_validation (1.1.2)
-    htmlentities (4.3.3)
+    htmlentities (4.3.4)
```
Comment (rnalexander) [inline Gemfile.lock]: Again, why is this going up?

File: Gemfile.lock [hunk 3]
```diff
@@ -378,13 +379,13 @@ GEM
...
     http-cookie (1.0.2)
       domain_name (~> 0.5)
     i18n (0.7.0)
     i18n-js (3.0.0.mas)
       i18n
-    icalendar (2.2.2)
+    icalendar (2.3.0)
```
Comment (rnalexander) [inline Gemfile.lock]: And again

File: Gemfile.lock [hunk 4]
```diff
@@ -280,11 +281,11 @@ GEM
       sass-rails (~> 4.0.3)
       syslog-logger
       uglifier (>= 1.0.3)
-    delayed_job (4.0.6)
+    delayed_job (4.1.1)
       activesupport (>= 3.0, < 5.0)
-    delayed_job_active_record (4.0.3)
-      activerecord (>= 3.0, < 5.0)
-      delayed_job (>= 3.0, < 4.1)
+    delayed_job_active_record (4.1.0)
```
Comment (rnalexander) [inline Gemfile.lock]: And again.

File: Gemfile.lock (hunk 2 above)
Comment (benbarnett) [inline Gemfile.lock]: These are just loosely pinned to the minor version, usually acceptable to update

**Comment by rnalexander** [comment]
There's a LOT of changes of things in the gemfile.lock that seem to have either been triggered by the updating of these modules.  That's a bit concerning.

**Comment by benbarnett** [comment]
@johnplayer1982 was this definitely done from specifically doing `bundle update <engine name>` rather than just a `bundle update`? I think most of the updates are fine though as they're quite loosely pinned dependencies, but worth double checking

+ @rnalexander

**Comment by johnplayer1982** [comment]
Yes I done bundle update <engine name> for each of the engines which were changes for this task

File: Gemfile.lock [hunk 5]
```diff
@@ -785,6 +786,3 @@ DEPENDENCIES
   validate_url (= 1.0.0)
   vcr
   webmock
-
-BUNDLED WITH
-   1.10.5
```
Comment (tomas-stefano) [inline Gemfile.lock]: I think this was a problem with bundle update. What exactly the gem that you want update?

**Comment by Tr4pSt3R** [comment]
Changing the canonical partial template files should require no need to update the Gemfile. Strip out the changes to the Gemfile and you are good to go. 
@johnplayer1982 give me a shout if you are available to pair up on this

**Comment by benbarnett** [comment]
@Tr4pSt3R but there are changes to the engines that need to be updated too

**Comment by johnplayer1982** [comment]
@tomas-stefano @Tr4pSt3R  I only needed to update the versions of the engines which were changed as part of this task.  Any uneccessary changes to the Gemfile.lock weren't intentional, whats the solution for this?

**Comment by Tr4pSt3R** [comment]
@benbarnett I understand what you mean now. 
@rnalexander: The Gemfile changes are legitimate as changes occurred to the MAS engines listed below.
@johnplayer1982 FYI all seems to be okay

feedback https://github.com/moneyadviceservice/feedback/pull/6
contribution_calculator https://github.com/moneyadviceservice/contribution_calculator/pull/3
payday_loans_intervention https://github.com/moneyadviceservice/payday_loans_intervention/pull/46
baby_cost_calculator https://github.com/moneyadviceservice/baby_cost_calculator/pull/5
timelines https://github.com/moneyadviceservice/timelines/pull/60

**Comment by rnalexander** [comment]
cool cool.  I'll bring it in for the next round of pushing.

**Comment by johnplayer1982** [comment]
Cool thanks all

**Comment by rnalexander** [comment]
This is now on QA.

**Pull request merged**